signer/standard_signer.go

```diff
@@ -223,7 +225,8 @@ func (s *StandardSigner) Sign(hostName string, in []byte, subject *Subject, prof
 
 	template, err := ParseCertificateRequest(s, block.Bytes, subject)
 	if err != nil {
-		return
+		return nil, cferr.New(cferr.CertificateError,
+			cferr.DecodeFailed, errors.New("not a certificate or csr"))
```

Comment:
This error does not need to be wrapped, ParseCertificateRequest returns a cfssl error.